Makefile

```diff
@@ -19,6 +19,11 @@ build-deprecated:
 check-fedora-versions:
 	`${COMPARE_FEDORA_VERSIONS} | jq '[.missing == ["libblkid-rs-sys"], .high == []] | all'`
 
+check-fedora-versions-sys:
+	`${COMPARE_FEDORA_VERSIONS} \
+	--manifest-path=./libblkid-rs-sys/Cargo.toml \
+	| jq '[.missing == ["libblkid-rs-sys"], .high == []] | all'`
```

Comment:
Bet you can drop "libblkid-rs-sys" in this line. It ought not to depend on itself.

Reply:
Confirmed; it works after removing it.